### PR TITLE
Feature: Add support for multiple service in the same spec

### PR DESCRIPTION
**Pull request by timotheeguerin**
fix https://github.com/microsoft/cadl/issues/1052

Breaking change entry https://github.com/microsoft/cadl/issues/1263#issuecomment-1320267176

### CODE REVIEW

**Comment by bterlson** [comment]
Can you summarize the changes required?

**Comment by timotheeguerin** [comment]
> Can you summarize the changes required?

Writing the breaking change description here https://github.com/microsoft/cadl/issues/1263#issuecomment-1320267176 (not fiinished yet)

**Comment by bterlson** [comment]
Can we leave the old methods there that are deprecated and just do the "no-think replacement" you have in the changelog?

**Review by bterlson — APPROVED** [review]
